frontend/src/components/KeyMeasureDiagnostic/ExpeModals/ExpeVegetarian.vue

```diff
@@ -0,0 +1,788 @@
+<template>
+  <div>
+    <div v-if="showSpinner" style="min-height: 290px; background-color: white">
+      <v-progress-circular indeterminate style="position: absolute; left: 50%; top: 50%"></v-progress-circular>
+    </div>
+    <v-card class="text-left" v-else-if="expe">
+      <v-card-title class="font-weight-black">
+        Vous êtes inscrit à l'expérimentation
+      </v-card-title>
+      <v-card-text>
+        <p>
+          Afin d'évaluer les effets de la mise en place du menu végétarien quotidien sur la satisfaction des usagers et
+          le taux de fréquentation, nous vous demandons de suivre et renseigner différents indicateurs ci-dessous à deux
+          étapes de l’expérimentation.
+        </p>
+        <p>
+          Pour plus d'informations, consultez le
+          <a
+            href="https://ma-cantine-1.gitbook.io/ma-cantine-egalim/diversification-des-sources-de-proteines-et-menus-vegetariens/guide-pour-la-mise-en-place-du-menu-vegetarien-en-milieu-scolaire"
+            target="_blank"
+          >
+            livret de recettes végétariennes du CNRC
+          </a>
+          et le
+          <a
+            href="https://ma-cantine-1.gitbook.io/ma-cantine-egalim/diversification-des-sources-de-proteines-et-menus-vegetariens/untitled"
+            target="_blank"
```

Comment:
For both of these links: add rel="noopener" and an external link icon like <v-icon color="primary" x-small>mdi-open-in-new</v-icon> ?

Reply:
Good idea : https://github.com/betagouv/ma-cantine/pull/1237/commits/8b186a2d990eab5def5b72d209636ddf8a475c96